cogboard-webapp/src/components/Widget/index.js

```diff
@@ -45,7 +45,7 @@ const Widget = ({ id, index }) => {
     ...widgetTypeData
   } = widgetData;
   const { expandContent } = widgetTypeData;
-  const widgetTypeConfig = widgetTypes[type];
+  const widgetTypeConfig = getWidgetType(type);
```

Comment:
`const widgetTypeConfig = widgetTypes[type] | widgetTypes['WhiteSpaceWidget'];`

Reply:
True, I though about this. In my opinion, introducing helper function will prevent developers from using `widgetTypes[type]` explicitly.

Reply:
Done.